src/Commands/CreateSuperAdmin.php

```diff
@@ -12,41 +12,87 @@ class CreateSuperAdmin extends Command
 
     protected $description = "Create the superadmin account";
 
+    /**
+     * Create super admin account.
+     *
+     * @return void
+     */
     public function handle()
     {
         $this->info("Let's create a superadmin account!");
+        $email = $this->setEmail();
+        $password = $this->setPassword();
+        User::create([
+            'name' => "Admin",
+            'email' => $email,
+            'password' => bcrypt($password),
+            'role' => 'SUPERADMIN',
+            'published' => true,
+        ]);
+        $this->info("Your account has been created");
+    }
+
+    /**
+     * Prompt user to enter email and validate it.
+     *
+     * @return string $email
+     */
+    private function setEmail()
+    {
         $email = $this->ask('Enter an email');
         if ($this->validateEmail($email)) {
-            $password = $this->ask('Enter a password');
-            if ($this->validatePassword($password)) {
-                return User::create([
-                    'name' => "Admin",
-                    'email' => $email,
-                    'password' => bcrypt($password),
-                    'role' => 'SUPERADMIN',
-                    'published' => true,
-                ]);
-            } else {
-                $this->error("Your password is not valid");
-            }
+            return $email;
         } else {
             $this->error("Your email is not valid");
+            $this->setEmail();
         }
+    }
 
-        $this->info("Your account has been created");
+    /**
+     * Prompt user to enter password, confirm and validate it.
+     *
+     * @return string $password
+     */
+    private function setPassword()
+    {
+        $password = $this->secret('Enter a password');
+        if ($this->validatePassword($password)) {
+            $confirmPassword = $this->secret('Confirm the password');
+            if ($password === $confirmPassword) {
+                return $password;
+            } else {
+                $this->error('Password does not match the confirm password');
+                $this->setPassword();
+            }
+        } else {
+            $this->error("Your password is not valid, at least 6 characters");
+            $this->setPassword();
+        }
     }
 
+    /**
+     * Determine if the email address given valid.
+     *
+     * @param  string  $email
+     * @return boolean
+     */
     private function validateEmail($email)
     {
-        return Validator::make(['email' => $email], [
-            'email' => 'required|email|max:255|unique:users',
-        ]);
+        return !Validator::make(['email' => $email], [
+            'email' => 'required|email|max:255|unique:twill_users',
+        ])->fails();
```

Comment:
Hey @yanhao-li, I'm ready to merge this is, it is working great :)

Would you mind updating validate functions like the following? (removing the not operator in front obviously, which I'm suggesting in another comment, since the new Github feature to suggest changes on PR does not allow multi line suggestions yet!)

```suggestion
        ])->passes();
```